src/lib/eina/eina_main.c

```diff
@@ -296,7 +304,8 @@ eina_init(void)
 #endif
 
 #ifdef EFL_HAVE_THREADS
-   _eina_main_loop = pthread_self();
+   //_eina_main_loop = pthread_self();
+   _eina_main_loop = eina_thread_self();
```

Comment:
I did the same change here, merge conflict alert

Reply:
This can be solved during merge process, don't worry :)